.github/workflows/test.yml

```diff
@@ -33,10 +35,10 @@ jobs:
           miniforge-variant: Mambaforge
           miniforge-version: latest
           use-mamba: true
+          channel-priority: strict
+          environment-file: continuous_integration/environment-${{ matrix.environment }}.yml
           activate-environment: test
-          condarc-file: continuous_integration/condarc
-          python-version: ${{ matrix.python-version }}
-          environment-file: continuous_integration/environment.yml
```

Comment:
Work around bug in miniforge github action where python-version + parameterized environment-file breaks Windows and MacOSX